make it possible to suppress solver output when running CBC and GLPK

maurerle opened the pull request:
Currently, the solver output is printed all the time to stdout using `print` when using CBC or GLPK.
This should be fixed, by also using the logging module.

I did set this to INFO, so that users relying on this output can still use it without going to debug logs.

It would be very cool to have release with this change afterwards

maurerle commented:
It does not look like I can do anything about 3.9-macos and 3.11-macos failing - it says `Fatal Python error: Illegal instruction` coming from `pyscipopt` solving with `scip`

FabianHofmann commented:
Great! thanks @maurerle, would you mind putting a short note to the release notes? CI problem seemed to be due to a wrong installation (is working now).

maurerle commented:
Sure, is it fine like this @FabianHofmann ?

FabianHofmann commented:
awesome, thanks!

maurerle commented:
@FabianHofmann could you please publish a release with this version soon?

FabianHofmann commented:
@maurerle already happened :) conda should be available within the next hour

maurerle commented:
Would you also add the release here:
https://github.com/PyPSA/linopy/releases
This is also missing for 0.3.3

This would make it possible to easily see the changes since the last release :)
